src/openforms/submissions/models.py

```diff
@@ -511,8 +513,24 @@ def _get_value_label(possible_values: list, value: str) -> str:
         for possible_value in possible_values:
             if possible_value["value"] == value:
                 return possible_value["label"]
+        # TODO what if value is not a string? shouldn't it be passed through something to covert for display?
         return value
 
+    @staticmethod
+    def _get_values(value, filter_func=bool):
+        # filter a single or multiple value into a list of acceptable values
+        if isinstance(value, list):
+            return list(filter(filter_func, value))
+        elif not filter_func(value):
+            return []
+        else:
+            return [value]
+
+    @classmethod
+    def _join_mapped(cls, func, value, seperator=", "):
+        # filter and map a single or multiple value into a joined string
+        return seperator.join(map(func, cls._get_values(value)))
```

Comment:
`filter` and `map` aren't very pythonic - there are some voices that even claim it was a mistake to add them to Python.

can you rewrite these as list comprehensions instead - that's usually more readable and pythonic

Reply:
Additionally, these two static/class methods seem to be used only in a single place - can't we just make these module-level helper functions instead of static/class methods? I don't mind fat models, but this is getting very bloated.